Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -9,6 +9,64 @@
 
 namespace Hazel {
 
+	namespace InstrumentorUtils {
+
+		template <size_t N>
+		struct ChangeResult
+		{
+			char Data[N];
+		};
+
+		template <size_t N>
+		constexpr auto RemoveCdecl(const char(&expr)[N])
+		{
+			ChangeResult<N> result = {};
+
+			size_t srcIndex = 0;
+			size_t dstIndex = 0;
+			while (srcIndex < N - 2) {
+				if (expr[srcIndex]     == '_' &&
+					expr[srcIndex + 1] == '_' &&
+					expr[srcIndex + 2] == 'c' &&
+					expr[srcIndex + 3] == 'd' &&
+					expr[srcIndex + 4] == 'e' &&
+					expr[srcIndex + 5] == 'c' &&
+					expr[srcIndex + 6] == 'l' &&
+					expr[srcIndex + 7] == ' ')
+				{
+					srcIndex += 8;
+				}
+				result.Data[dstIndex++] = expr[srcIndex++];
+			}
+			result.Data[dstIndex++] = expr[N - 2];
+			result.Data[dstIndex++] = expr[N - 1];
+			return result;
+		}
+
+		template <size_t N>
+		constexpr auto ReplaceQuotes(const char(&expr)[N])
+		{
+			ChangeResult<N> result = {};
```

Comment:
Initializer list is not needed?

```suggestion
			ChangeResult<N> result;
```